Sketch.java

```diff
@@ -1,35 +1,144 @@
 import processing.core.PApplet;
 
 public class Sketch extends PApplet {
-	
-	
-  /**
-   * Called once at the beginning of execution, put your size all in this method
-   */
+	int w = 900;
+  int h = 900;
+	float a = random(0, w);
+  float b = random(0, h);
+  float c = random(0, w);
+  float d = random(0, h);
+  float e = random(0, w);
+  float f = random(0, h);	
+  float g = random(0, w);
+  float i = random(0, h);
+  float j = random(0, w);
+  float k = random(0, h);
+  float l = random(0, w);
+  float m = random(0, h);
+  float n = random(0, w);
+  float o = random(0, h);
+  float p = random(0, w);
+  float q = random(0, h);
+  float r = random(0, w);
+  float s = random(0, h);
+  float t = random(0, w);
+  float u = random(0, h);
+ 
+  
+ 
+  
+  
   public void settings() {
 	// put your size call here
-    size(400, 400);
-  }
+    size(w, h);
+}
 
   /** 
    * Called once at the beginning of execution.  Add initial set up
    * values here i.e background, stroke, fill etc.
    */
   public void setup() {
-    background(210, 255, 173);
+    background(135,206,235);
   }
 
   /**
    * Called repeatedly, anything drawn to the screen goes here
    */
   public void draw() {
```

Comment:
although values are being generated randomly, it results in the drawing of a non-discerning object as indicated in your submitted screenshot.